TASK-54423 Implement new Extensible Register UI and Onboarding Form

boubaker opened the pull request:
This enhancement allows to enable user registration using onboarding process.

boubaker committed:
TASK-54423 Implement new Extensible Register UI and Onboarding Form
This enhancement allows to enable user registration using onboarding process.

ghost commented:
### CodeSee Review Map:
[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/Meeds-io/social/1314/c396fb1b/2754d513419dca9075c6df230229e8fca3eee93e.svg)](https://app.codesee.io/r/reviews?pr=1314&src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

Review [in an interactive map](https://app.codesee.io/r/reviews?pr=1314&src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

View [more CodeSee Maps](https://app.codesee.io/r/maps?src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

#### Legend
![CodeSee Map Legend](https://app.codesee.io/Legend_v2.png)

boubaker committed:
TASK-54423 Allow to define a header and footer for extension-registry-components
Prior to this change, the component 'extension-registry-components' wasn't allowing to add a separator, footer or header in extend components. This change uses slots to allow define custom content for separator, footer and header for Login & Register screens
